Manifests/ManifestsApple/com.apple.applicationaccess-macOS.plist

```diff
@@ -255,7 +261,9 @@ A profile can consist of payloads with different version numbers. For example, c
 				<key>pfm_description</key>
 				<string/>
 				<key>pfm_description_reference</key>
-				<string>Optional. When false, this disallows content caching. Defaults to true. Availability: Available only in macOS 10.13 and later.</string>
+				<string>Optional. When false, this disallows content caching. Defaults to true. Introduced in macOS 10.13 but moved to the content caching domain in 10.13.4.</string>
```

Comment:
I think that `pfm_description` fits this value better than `pfm_description_reference` because the text is not reference text. Therefore I simply changed the tag and reworded slightly. See 68a1f49.